functions/Backup-DbaDatabase.ps1

```diff
@@ -71,27 +75,47 @@ Backs up AdventureWorks2014 to sql2016's C:\temp folder
 #>
 	[CmdletBinding(DefaultParameterSetName = "Default")]
 	param (
-		[parameter(ParameterSetName = "NoPipe", Mandatory = $true)]
+		[parameter(ParameterSetName = "NoPipe", Mandatory = $true)][parameter(ParameterSetName = "Pipe")]
```

Comment:
If someone is piping in full on SMO objects, then they'll have their connected server inside the object, so $server = $database.parent thus they won't need -SqlInstance. That's why $Sqlinstance is not in the Pipe ParameterSetName.